tools/create-cs.sh

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
+tag="$1"
+
+[ $# -eq 0 ] && { echo "Usage: $0 <cincinnati-operator-registry image tag>"; exit 1; }
```

Comment:
I'm used to seeing placeholder in caps in the usage, instead of in angle-brackets.  How about:

```sh
if test 1 -ne $#
then
  echo "Usage: $0 OPERATOR_IMAGE_TAG" >&2
  exit 1
fi
```

Reply:
changed